templates/zerver/app/compose.html

```diff
@@ -96,8 +96,8 @@
                                 <div id="below-compose-content">
                                     <input type="file" id="file_input" class="notvisible pull-left" multiple />
                                     <a class="message-control-button fa fa-smile-o" aria-hidden="true" id="emoji_map" href="#" title="{{ _('Add emoji') }}"></a>
-                                    <a class="message-control-button fa fa-font" aria-hidden="true" title="{{ _('Formatting') }}" data-overlay-trigger="message-formatting"></a>
-                                    <a class="message-control-button fa fa-paperclip notdisplayed" aria-hidden="true" id="attach_files" href="#" title="{{ _('Attach files') }}"></a> {% if jitsi_server_url %}
+                                    <a class="message-control-button fa fa-font" aria-hidden="true" title="{{ _('Formatting') }}" data-overlay-trigger="message-formatting"></a> {% if max_file_upload_size > 0 %}
```

Comment:
The linewrapping here is weird; let's put the conditionals on their own lines.

Also, you should do this same change in the message edit templates.

Reply:
I forgot about the message edit templates. My bad! Made the appropriate changes there, and also fixed the line-wrapping here. 
Thank you for pointing it out